lib/dfe_sign_in/api/user.rb

```diff
@@ -22,19 +66,7 @@ def role_codes
       private
 
       def body
-        @body ||= begin
-          request = Net::HTTP::Get.new(uri)
-          request["Authorization"] = "bearer #{generate_jwt_token}"
-          request["Content-Type"] = "application/json"
-
-          response = Net::HTTP.start(uri.hostname, uri.port, use_ssl: true) { |http|
-            http.request(request)
-          }
-
-          raise ExternalServerError, "#{response.code}: #{response.body}" unless response.code.eql?("200")
-
-          JSON.parse(response.body)
-        end
+        @body ||= self.class.get(uri)
```

Comment:
because the module with the `get` method has been included and extended, could this be written as:

```suggestion
        @body ||= get(uri)
```